clientd3d/makefile

```diff
@@ -150,6 +159,9 @@ $(OUTDIR)\crc.obj : $(TOPDIR)\util\crc.c
 $(OUTDIR)\md5.obj : $(TOPDIR)\util\md5.c
 	$(CC) -Fo$(OUTDIR)/ $(CFLAGS) /Fd$(OUTDIR)\vc90.pdb /FpCpch -c $**
 
+$(OUTDIR)\stb_vorbis.obj : $(SOURCEDIR)\stb_vorbis.c
```

Comment:
I think it would be less duplication if you just added -DSTB_VORBIS_NO_PUSHDATA_API to CFLAGS, rather than a whole new rule for this one source file.  It's not like any other code is going to be affected by that symbol.